src/plugins/saved_objects_management/public/management_section/objects_table/components/flyout.tsx

```diff
@@ -189,13 +190,21 @@ export class Flyout extends Component<FlyoutProps, FlyoutState> {
    * Does the initial import of a file, resolveImportErrors then handles errors and retries
    */
   import = async () => {
-    const { http } = this.props;
+    const { http, workspaces } = this.props;
     const { file, importMode, selectedDataSourceId } = this.state;
     this.setState({ status: 'loading', error: undefined });
 
     // Import the file
     try {
-      const response = await importFile(http, file!, importMode, selectedDataSourceId);
+      const response = await importFile(
+        http,
+        file!,
+        {
+          ...importMode,
+          workspaces,
```

Comment:
Seems this `workspaces` passed from `this.props` will override the `importMode.workspaces`, should it be the other way around?